codalab/model/bundle_model.py

```diff
@@ -1907,8 +1907,12 @@ def batch_get_user_in_group(self, **kwargs):
                 return []
         return [str_key_dict(row) for row in rows]
 
-    # Helper function: return list of group uuids that |user_id| is in.
-    def _get_user_groups(self, user_id):
+    def get_user_groups(self, user_id):
```

Comment:
Interesting that we already supported groups earlier. What were groups used for, if not for sharing workers?

Reply:
Groups are also used for worksheets and bundles.